### PR TITLE
Import all content items

### PR DESCRIPTION
**Pull request by leenagupte**
Trello card: https://trello.com/c/PjAzxNQS

## Motivation

As the application is currently in production, we need to load all content items for all organisations.

This will allow us to:

1. Distribute the application to other users within GDS
2. Discover performance issues in advance

### CODE REVIEW

File: app/models/importers/all_content_items.rb [hunk 1]
```diff
@@ -1,6 +1,12 @@
 module Importers
   class AllContentItems
     def run
+      organisations = ::Organisation.pluck(:slug)
```
Comment (pmanrubia) [inline app/models/importers/all_content_items.rb]: This is loading all organisations into memory.
I would go for something like:

```ruby
importer = ContentItemsByOrganisation.new
Organisation.find_each do |organisation| 
  importer.run(organisation.slug)
end
```

Comment (leenagupte) [inline app/models/importers/all_content_items.rb]: Thanks. I'll give it a go and make the change.

Comment (leenagupte) [inline app/models/importers/all_content_items.rb]: @pmanrubia Updated and fixed up in commit 5a48bb5

**Comment by pmanrubia** [comment]
👍

**Pull request merged**